### PR TITLE
 [CI:DOCS]  Unify podman prune filter description: volumes, networks, system

### PR DESCRIPTION
**Pull request by patrycja-guzik**
follow up of #12976

Signed-off-by: Patrycja Guzik <patrycja.k.guzik@gmail.com>

### CODE REVIEW

File: docs/source/markdown/podman-network-prune.1.md [hunk 1]
```diff
@@ -18,17 +18,20 @@ Do not prompt for confirmation
...
+Supported filters:
+
+| Filter             | Description                                                                 |
+| :----------------: | --------------------------------------------------------------------------- |
+| *until*            | Only remove networks created before given timestamp.           |
+| *label*            | Only remove networks, with (or without, in the case of label!=[...] is used) the specified labels.                  |
+
+The `until` *filter* can be Unix timestamps, date formatted timestamps, or Go duration strings (e.g. 10m, 1h30m) computed relative to the machine’s time.
+
+The `label` *filter* accepts two formats. One is the `label`=*key* or `label`=*key*=*value*, which removes networks with the specified labels. The other format is the `label!`=*key* or `label!`=*key*=*value*, which removes networks without the specified labels.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-network-prune.1.md:34]: I realize it's only two entries, but I'd rather these be kept in alphanumeric order, both in the table and then at lines 32 and 34.  The text looks great though

File: docs/source/markdown/podman-system-prune.1.md [hunk 2]
```diff
@@ -22,16 +22,18 @@ Recursively remove all unused pod, container, image and volume data (Maximum 50
...
+| *until*            | Only remove containers and images created before given timestamp.           |
+| *label*            | Only remove containers and images, with (or without, in the case of label!=[...] is used) the specified labels.                  |
 
-The until filter can be Unix timestamps, date formatted timestamps, or Go duration strings (e.g. 10m, 1h30m) computed relative to the machine’s time.
+The `until` *filter* can be Unix timestamps, date formatted timestamps, or Go duration strings (e.g. 10m, 1h30m) computed relative to the machine’s time.
 
-The label filter accepts two formats. One is the label=... (label=_key_ or label=_key=value_), which removes containers and images with the specified labels. The other format is the label!=... (label!=_key_ or label!=_key=value_), which removes containers and images without the specified labels.
+The `label` *filter* accepts two formats. One is the `label`=*key* or `label`=*key*=*value*, which removes containers and images with the specified labels. The other format is the `label!`=*key* or `label!`=*key*=*value*, which removes containers and images without the specified labels.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-system-prune.1.md:36]: ditto ordering comment.

File: docs/source/markdown/podman-volume-prune.1.md [hunk 3]
```diff
@@ -21,12 +21,20 @@ Do not prompt for confirmation.
...
+Supported filters:
+
+| Filter             | Description                                                                 |
+| :----------------: | --------------------------------------------------------------------------- |
+| *until*            | Only remove volumes created before given timestamp.           |
+| *label*            | Only remove volumes, with (or without, in the case of label!=[...] is used) the specified labels.                  |
+
+The `until` *filter* can be Unix timestamps, date formatted timestamps, or Go duration strings (e.g. 10m, 1h30m) computed relative to the machine’s time.
+
+The `label` *filter* accepts two formats. One is the `label`=*key* or `label`=*key*=*value*, which removes volumes with the specified labels. The other format is the `label!`=*key* or `label!`=*key*=*value*, which removes volumes without the specified labels.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-volume-prune.1.md:37]: ditto ordering comment.

**Comment by TomSweeneyRedHat** [comment]
Thanks @patrycja-guzik , just one generic comment to keep my OCD in check.

**Comment by rhatdan** [comment]
/approve
LGTM
Thanks @patrycja-guzik 
Fix the ordering and we can merge.

**Comment by rhatdan** [comment]
Thanks @patrycja-guzik I made the changes @TomSweeneyRedHat suggested and fixed it in a couple of other existing man pages in #13034

**Comment by patrycja-guzik** [comment]
@rhatdan, thank you for fixing it!

**Comment by rhatdan** [comment]
No problem. Thanks for your PR.

**Pull request closed without merging**